packages/liveblocks-react/src/room.tsx

```diff
@@ -2402,48 +2583,326 @@ export function generateQueryKey(
   return `${roomId}-${stringify(options ?? {})}`;
 }
 
-type DefaultRoomContextBundle = RoomContextBundle<DP, DS, DU, DE, DM>;
-
-const _RoomProvider: DefaultRoomContextBundle["RoomProvider"] = RoomProvider;
-const _useBroadcastEvent: DefaultRoomContextBundle["useBroadcastEvent"] =
-  useBroadcastEvent;
-const _useOthersListener: DefaultRoomContextBundle["useOthersListener"] =
-  useOthersListener;
-const _useRoom: DefaultRoomContextBundle["useRoom"] = useRoom;
-const _useAddReaction: DefaultRoomContextBundle["useAddReaction"] =
-  useAddReaction;
-const _useMutation: DefaultRoomContextBundle["useMutation"] = useMutation;
-const _useCreateThread: DefaultRoomContextBundle["useCreateThread"] =
-  useCreateThread;
-const _useDeleteThread: DefaultRoomContextBundle["useDeleteThread"] =
-  useDeleteThread;
-const _useEditThreadMetadata: DefaultRoomContextBundle["useEditThreadMetadata"] =
+type TypedBundle = RoomContextBundle<DP, DS, DU, DE, DM>;
+
+/**
+ * Makes a Room available in the component hierarchy below.
+ * Joins the room when the component is mounted, and automatically leaves
+ * the room when the component is unmounted.
+ */
+const _RoomProvider: TypedBundle["RoomProvider"] = RoomProvider;
+
+/**
+ * Returns a callback that lets you broadcast custom events to other users in the room
+ *
+ * @example
+ * const broadcast = useBroadcastEvent();
+ *
+ * broadcast({ type: "CUSTOM_EVENT", data: { x: 0, y: 0 } });
+ */
+const _useBroadcastEvent: TypedBundle["useBroadcastEvent"] = useBroadcastEvent;
+
+/**
+ * Get informed when users enter or leave the room, as an event.
+ *
+ * @example
+ * useOthersListener({ type, user, others }) => {
+ *   if (type === 'enter') {
+ *     // `user` has joined the room
+ *   } else if (type === 'leave') {
+ *     // `user` has left the room
+ *   }
+ * })
+ */
+const _useOthersListener: TypedBundle["useOthersListener"] = useOthersListener;
+
+/**
+ * Returns the Room of the nearest RoomProvider above in the React component
+ * tree.
+ */
+const _useRoom: TypedBundle["useRoom"] = useRoom;
+
+/**
+ * Returns a function that adds a reaction from a comment.
+ *
+ * @example
+ * const addReaction = useAddReaction();
+ * addReaction({ threadId: "th_xxx", commentId: "cm_xxx", emoji: "👍" })
+ */
+const _useAddReaction: TypedBundle["useAddReaction"] = useAddReaction;
+
+/**
+ * Create a callback function that lets you mutate Liveblocks state.
+ *
+ * The first argument that gets passed into your callback will be
+ * a "mutation context", which exposes the following:
+ *
+ *   - `root` - The mutable Storage root.
```

Comment:
I think this is `storage` not `root`. It's mentioned a couple of other times in this JSDoc comment too.

Reply:
Ah, good catch. These comments are all blindly copy-pasted from their equals, so I didn't read them through during the process. Fixed in ba88ab8bdfddf48080af0186ac00325ce76762fe 👍